src/Collection/CollectionBuilder.php

```diff
@@ -83,7 +83,7 @@ public function __construct($commandFile)
     }
 
     /**
-     * @param \League\Container\ContainerInterface $container
+     * @param \League\Container\Container $container
```

Comment:
Shouldn't all of these references to ContainerInterface become Psr\Container\ContainerInterface?

Reply:
I was thinking that and even made that change in a local branch based on 2.x at one point but wasn't totally sure since Robo uses some `League\Container` specific features.  I'll try making that change to this branch now.